### ISSUE TITLE
missing field in structure, should give syntax error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2148]
field `something_very_wrong_bad` is not part of the method input structure,

```abap
CLASS lcl_bar DEFINITION.
  PUBLIC SECTION.
    TYPES: BEGIN OF ty_type,
             field TYPE i,
           END OF ty_type.
    METHODS run.
    METHODS bar IMPORTING foo TYPE ty_type.
ENDCLASS.

CLASS lcl_bar IMPLEMENTATION.
  METHOD run.
    bar( VALUE #( something_very_wrong_bad = 2 ) ).
  ENDMETHOD.

  METHOD bar.
    RETURN.
  ENDMETHOD.
ENDCLASS.
```
